src/features/events/layout/EventLayout.tsx

```diff
@@ -27,12 +31,20 @@ const EventLayout: React.FC<EventLayoutProps> = ({
   orgId,
   campaignId,
 }) => {
+  const [showBorder, setShowBorder] = useState<boolean>(false);
+  const [autoOnEdit, setAutoOnEdit] = useState<boolean>(false);
```

Comment:
What is the purpose of the `autoOnEdit` state, and how is it different from the `showBorder` state?

Reply:
I found that when user keep clicking title - autocomplete - title autocomplete repeatedly, it would disable border from event title cause `ZUiEditTextInPlace` doesn't know if autocomplete is still in edit mode or not. That was the reason why